ES/ESdd041.xml

```diff
@@ -177,144 +184,131 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                           colophon; elements of the punctuation signs; Ethiopic numerals or their elements.</seg>
                      </handNote>
                   </handDesc>
                   <decoDesc>
 
                      <decoNote type="frame" xml:id="d1">
-                        <locus target="#3r">3r</locus> (incipit page of Text I): elaborate, coloured
-                        (red) ornamental band, with two lateral and one central extensions reaching
-                        l.7; interlaces.</decoNote>
+                        <locus target="#3r"/> 
+                        <desc>Elaborate, coloured
+                           (red) ornamental band, with two lateral and one central extensions reaching <locus target="#3r7"/>;
+                           interlaces (incipit page of <ref target="ms_i1.1"/>).</desc>
+                     </decoNote>
                   </decoDesc>
+                  
                   <additions>
                      <list>
                         <item xml:id="a1">
                            <locus target="#104rb"/>
-                           <desc type="GuestText">Historiographic note on the time of King Śarḍa Dǝngǝl. </desc>
+                           <desc type="Unclear">Historiographic note on the time of  <persName ref="PRS8550sardaDe"><roleName type="title">King</roleName> Śarḍa Dǝngǝl</persName>. </desc>                           
                            <q xml:lang="gez">
-                              ወነግሠ፡ ንጉሥ፡ መለክ፡ ሰገድ፡ እንዘ፡ ወልደ፡
-                              ፲ወ፫፡ ዓመት፡ ወ፯፡ አውራኅ። ወበ፭፡ ዓመት፡ ሞተት፡ ኢቴ፡ ሰብለ፡ ወንጌል፤ ወበ፲ወ፪፡ ዓመት፡ ሞተ፡
-                              አዝማች፡ ሀርቦ። ወበ፲ወ፬፡ {ዓመት፡}ሞተ፡ መሐመድ፡ ወበ፲ወ፯፡ ዓመት፡ ሞተ፡ አዝማች፡ ይስሐቅ። ወበ፵ወ፩፡
-                              ዓመት፡ አዕረፈ፡ በምድረ፡ ሻች።</q>
+                              ወነግሠ፡ <persName ref="PRS8550sardaDe"> <roleName type="title">ንጉሥ፡</roleName> መለክ፡ ሰገድ፡</persName>  እንዘ፡ ወልደ፡
+                              ፲ወ፫፡ ዓመት፡ ወ፯፡ አውራኅ። ወበ፭፡ ዓመት፡ ሞተት፡ <persName ref="PRS8276SablaWa"><roleName type="title">ኢቴ፡</roleName> ሰብለ፡ ወንጌል፤</persName> ወበ፲ወ፪፡ ዓመት፡ ሞተ፡
+                              <persName ref="PRS5112harbo"><roleName type="title"> አዝማች፡</roleName> ሀርቦ</persName>። ወበ፲ወ፬፡ {ዓመት፡}ሞተ፡ <persName ref="PRS7278Muhammad"> መሐመድ፡</persName> ወበ፲ወ፯፡ ዓመት፡ ሞተ፡ <persName ref="PRS10353Yeshaq">
+                               <roleName type="title"> አዝማች፡</roleName> ይስሐቅ</persName>። ወበ፵ወ፩፡ ዓመት፡ አዕረፈ፡ በምድረ፡ ሻች።</q>
                            
-                           <note> The note is written in a hand which seems posterior to the main
-                              hand. It mentions King Malak [sic for Malāk] Sagad (i.e., Śarḍa Dǝngǝl, r. 1563-97) as well as ʾǝtege Sabla Wangel, ʾazmāč Ḥarbo,
-                              Maḥammad and ʾazmāč Yǝsḥaq. </note>
+                           <note> The note is written in a hand which seems posterior to the main<ref type="hand" corresp="h1"> hand</ref>. 
```

Comment:
to the main `<ref type="hand" corresp="h1">Hand 1</ref>` (attend spaces)

Reply:
good